scss/tech.scss

```diff
@@ -0,0 +1,57 @@
+
+.techs-container {
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.tech {
+  align-items:center;
+  background-color: $green;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  height: 100%;
+  margin: 20px;
+  padding: 10px 30px;
+  position: relative;
+  text-align: center;
+  width: $card-size;
+  z-index: 1;
+}
+
+.tech::before {
```

Comment:
Using SASS, you can use nesting classes. That helps readability and helps you override styles whenever necessary:

```scss
.tech {
  ...
  &::before {
    ..
  }
}
```

The `&` operator makes reference to the parent class